examples/webcam/webcam.py

```diff
@@ -66,14 +67,25 @@ async def on_connectionstatechange():
             pcs.discard(pc)
 
     # open media source
-    audio, video = create_local_tracks(args.play_from)
+    audio, video = create_local_tracks(
```

Comment:
if video and audio are Nones, you could raise an exception. For example, with `decode=False` is impossible to get video track (because decode is different of h264).